Vagrantfile

```diff
@@ -1,155 +1,146 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
+#
+
+LINUX_BASE_BOX = "bento/ubuntu-16.04"
+FREEBSD_BASE_BOX = "jen20/FreeBSD-11.1-RELEASE"
+
+Vagrant.configure(2) do |config|
+	# Compilation and development boxes
+	config.vm.define "linux", autostart: true, primary: true do |vmCfg|
+		vmCfg.vm.box = LINUX_BASE_BOX
+		vmCfg.vm.hostname = "linux"
+		vmCfg = configureProviders vmCfg,
+			cpus: suggestedCPUCores()
+
+		vmCfg = configureLinuxProvisioners(vmCfg)
+
+		vmCfg.vm.synced_folder '.',
+			'/opt/gopath/src/github.com/hashicorp/nomad'
+
+		vmCfg.vm.provision "shell",
+			privileged: true,
+			path: './scripts/vagrant-linux-unpriv-bootstrap.sh'
+	end
+
+	config.vm.define "freebsd", autostart: false, primary: false do |vmCfg|
+		vmCfg.vm.box = FREEBSD_BASE_BOX
+		vmCfg.vm.hostname = "freebsd"
+		vmCfg = configureProviders vmCfg,
+			cpus: suggestedCPUCores()
+
+		vmCfg.vm.synced_folder '.',
+			'/opt/gopath/src/github.com/hashicorp/nomad',
+			type: "nfs",
+			bsd__nfs_options: ['noatime']
+
+		vmCfg.vm.provision "shell",
+			privileged: true,
+			path: './scripts/vagrant-freebsd-priv-config.sh'
+
+		vmCfg.vm.provision "shell",
+			privileged: false,
+			path: './scripts/vagrant-freebsd-unpriv-bootstrap.sh'
+	end	
+
+	# Test Cluster (Linux)
+	1.upto(3) do |n|
+		serverName = "nomad-server%02d" % [n]
+		clientName = "nomad-client%02d" % [n]
+		serverIP = "10.199.0.%d" % [10 + n]
+		clientIP = "10.199.0.%d" % [20 + n]
+
+		config.vm.define serverName, autostart: false, primary: false do |vmCfg|
+			vmCfg.vm.box = LINUX_BASE_BOX
+			vmCfg.vm.hostname = serverName
+			vmCfg = configureProviders(vmCfg)
+			vmCfg = configureLinuxProvisioners(vmCfg)
+
+			vmCfg.vm.provider "virtualbox" do |_|
+				vmCfg.vm.network :private_network, ip: serverIP
+			end
+		
+			vmCfg.vm.synced_folder '.',
+				'/opt/gopath/src/github.com/hashicorp/nomad'
+	
+			vmCfg.vm.provision "shell",
+				privileged: true,
+				path: './scripts/vagrant-linux-priv-zeroconf.sh'
+		end
+		
+		config.vm.define clientName, autostart: false, primary: false do |vmCfg|
+			vmCfg.vm.box = LINUX_BASE_BOX
+			vmCfg.vm.hostname = clientName
+			vmCfg = configureProviders(vmCfg)
+			vmCfg = configureLinuxProvisioners(vmCfg)
+			
+			vmCfg.vm.provider "virtualbox" do |_|
+				vmCfg.vm.network :private_network, ip: clientIP
+			end
+			
+			vmCfg.vm.synced_folder '.',
+				'/opt/gopath/src/github.com/hashicorp/nomad'
+
+			vmCfg.vm.provision "shell",
+				privileged: true,
+				path: './scripts/vagrant-linux-priv-zeroconf.sh'
+		end
+	end
+end
+
+def configureLinuxProvisioners(vmCfg)
+	vmCfg.vm.provision "shell",
+		privileged: true,
+		inline: 'rm -f /home/vagrant/linux.iso'
```

Comment:
What's this? I don't seem to have that file in my current vagrant setup which uses the same box.

Reply:
This appears to be in `/home/vagrant` for the Bento boxes, at least on VMWare (which is what I'm using day to day since it supports nested virtualisation). I'm guessing it's a stray ISO of the VMWare tools. That said, I also think it's likely not supposed to be there hence calling it out explicitly here rather than putting it in a script.